src/Util/TestDox/TestDoxPrinter.php

```diff
@@ -191,21 +247,23 @@ protected function registerTestResult(Test $test, ?Throwable $t, int $status, fl
         $testName = $test instanceof Reorderable ? $test->sortId() : $test->getName();
 
         $result = [
-            'className'  => $this->formatClassName($test),
-            'testName'   => $testName,
-            'testMethod' => $this->formatTestName($test),
-            'message'    => '',
-            'status'     => $status,
-            'time'       => $time,
-            'verbose'    => $verbose,
+            'className'    => $this->formatClassName($test),
+            'testName'     => $testName,
+            'testMethod'   => $this->formatTestName($test),
+            'message'      => '',
+            'status'       => $status,
+            'time'         => $time,
+            'verbose'      => $verbose,
+            'suite'        => $this->currentSuiteName,
+            'dataprovider' => $this->currentSuiteIsDataProvider,
```

Comment:
What do you think about using consistent casing?

```suggestion
            'dataProvider' => $this->currentSuiteIsDataProvider,
```